public/locales/en-US/grafana.json

```diff
@@ -125,6 +125,25 @@
     },
     "save": "Save"
   },
+  "config-public-dashboard": {
+    "public-dashboard": {
+      "copy-button": "Copy",
+      "dashboard-url-field-label": "Dashboard URL",
+      "pause-sharing-dashboard": "Pause sharing dashboard",
+      "revoke-public-URL": "Revoke public URL",
+      "settings-title": "Settings"
+    }
+  },
+  "configuration": {
+    "public-dashboard": {
+      "default-time-range-label": "Default time range",
+      "default-time-range-label-desc": "The public dashboard uses the default time range settings of the dashboard",
+      "show-annotations": "Show annotations",
```

Comment:
nit: this key should be `show-annotations-label` to be consistent with its description.